packages/live-preview-sdk/src/inspectorMode/types.ts

```diff
@@ -61,6 +64,8 @@ export type InspectorModeMouseMoveMessage = {
 };
 export type InspectorModeTaggedElementsMessage = {
   elements: Array<InspectorModeElement>;
+  manualTagCount: number;
```

Comment:
NIT: should it be here manuallyTaggedCount? (and automaticallyTaggedCount)
we have a mix of manualTag and manuallyTagged 🤷

Reply:
I've adjust to `manuallyTaggedCount` and `automaticallyTaggedCount` for consistency

Reply:
I've adjusted